ocean_lib/assets/asset_downloader.py

```diff
@@ -52,8 +52,6 @@ def download_asset_files(
         raise AssertionError(
```

Comment:
commenting here since I can't comment on older code: once we get rid of the provider_uri arg, you can also skip the check for is_valid_provider. This will happen automatically as part of the build_service_endpoint inside DSP itself, not here.